inc/classes/subscriber/third-party/plugins/class-mobile-subscriber.php

```diff
@@ -221,6 +221,7 @@ public function delete_site_option_callback( $option, $network_id ) {
 	 */
 	public function maybe_update_mobile_cache_activation( $value, $old_value ) {
 		$plugins   = static::get_mobile_plugins();
+		$plugins   = array_keys( $plugins );
```

Comment:
Nice try, but no ^^'
https://3v4l.org/1t2FA

I think this is what happens here:
https://3v4l.org/ieCod

Reply:
The array you passed is not the correct one. Basically this one will be used:
https://3v4l.org/Y0IFQ

Reply:
Right, I was looking at the old version. Apologies.